tests/ui/spirv-attr/bad-infer-storage-class.stderr

```diff
@@ -19,7 +19,7 @@ warning: redundant storage class specifier, storage class is inferred from type
 9 |     #[spirv(uniform_constant)] warning: &Image!(2D, type=f32),
   |             ^^^^^^^^^^^^^^^^
 
-error: entry parameter type must be by-reference: `&spirv_std::image::Image<f32, spirv_std::image::Dimensionality::TwoD, spirv_std::image::ImageDepth::Unknown, spirv_std::image::Arrayed::False, spirv_std::image::Multisampled::False, spirv_std::image::Sampled::Unknown, spirv_std::image::ImageFormat::Unknown, core::option::Option::<spirv_std::image::AccessQualifier>::None>`
+error: entry parameter type must be by-reference: `&spirv_std::Image<f32, spirv_std::image::Dimensionality::TwoD, spirv_std::image::ImageDepth::Unknown, spirv_std::image::Arrayed::False, spirv_std::image::Multisampled::False, spirv_std::image::Sampled::Unknown, spirv_std::image::ImageFormat::Unknown, core::option::Option::<spirv_std::image::AccessQualifier>::None>`
```

Comment:
Why was this changed? It seems like an indication of something gone wrong.

Reply:
This is just because the Image struct is also re-exported at the root, because I was using a re-reexport to try to get the docs.

Since that didn't work anyway, this change should slide back out again when it changes to just a normal re-export.